bedrock/security/templates/security/bug-bounty.html

```diff
@@ -11,17 +11,49 @@ <h1 itemprop="name"><span class="highlight">Bug Bounty Program</span></h1>
     <div itemprop="articleBody">
       <h2>Introduction</h2>
 
-      <p>The Mozilla Security Bug Bounty Program is designed to encourage
-        security research in Mozilla software and to reward those who help us create
-        the safest Internet clients in existence.</p>
+      <p>The Mozilla Security Bug Bounty Program is designed to encourage security research in Mozilla software and to reward those who help us make the internet a safer place.</p>
 
-      <p>Many thanks to <a href="http://en.wikipedia.org/wiki/Linspire">Linspire</a>
-        and <a href="http://www.markshuttleworth.com/">Mark Shuttleworth</a>, who
-        provided start-up funding for this endeavor.</p>
+      <h2>General Eligibility</h2>
 
-      <p>Mozilla has paid out around 3 million dollars in bounties to our various researchers!</p>
+      <p>To be eligible for a reward under this program: </p>
+      
+      <ul class="prose">
+        <li>The security bug must be original and previously unreported.</li>
+        <li>The security bug must be a part of Mozilla’s code, not the code of a third party.  We will pay bounties for vulnerabilities in third-party libraries incorporated into shipped client code or third-party websites utilized by Mozilla.</li>
+        <li>You must not have written the buggy code or otherwise been involved in contributing the buggy code to the Mozilla project.</li>
+        <li>You must be old enough to be eligible participate in and receive payment from this program in your jurisdiction, or otherwise qualify to receive payment, whether through consent from your parent or guardian or some other way.</li>
+        <li>You must not be an employee, contractor, or otherwise have a business relationship with the Mozilla Foundation or any of its subsidiaries.</li>
+        <li>You should use your best effort not to access, modify, delete, or store user data or Mozilla’s data. Instead, use your own accounts or test accounts for security research purposes.</li>
+        <li>If you inadvertently access, modify, delete, or store user data, we ask that you notify Mozilla immediately at <a href="security@mozilla.org">security@mozilla.org</a> and delete any stored data after notifying us.</li>
```

Comment:
The link to the email address should have the `mailto:` protocol before the email address in the href.